setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     vega >= 3.5,<4
     websockets >= 10.0,<11
     xarray >=0.19,<2022.4
+    aiohttp >= 3.8,<4 # TODO: could replace websockets and requests
```

Comment:
Please use requests in a thread for now.
We could see in a different task if we remove requests and websockets

Reply:
https://github.com/geo-engine/geoengine-python/pull/119/commits/c6d1aa8fb1be94b7b1371734e5919ad3b9921d9a